### ISSUE TITLE
Touchpad jumping on 2020 MBP

### ISSUE THREAD

**Issue by ZProLegend007** [ZProLegend007/brunch-mac#11]

**Comment by DamianNiolet** [comment]
To be clear, it is only the touchpad and not a mouse with dongle.

**Comment by ZProLegend007** [comment]
Did this happen on r107/r109?

**Comment by DamianNiolet** [comment]
No, I don't recall it doing so, but I'd have to downgrade at this point to be sure.

**Comment by ZProLegend007** [comment]
All good, I'll keep that in mind while troubleshooting

**Comment by DamianNiolet** [comment]
I test it out on 118.19, still there.  And I wanted to clarify, it's only when clicking something, and not every time.  I going to install brunch 107 on the other 2020 I have at some point and see if maybe it's just this computer.

**Comment by ZProLegend007** [comment]
I found some logs in my linux OS relating to bcm5974 (the touchpad driver) finding touchpad jumps and ignoring them, perhaps for some reason this doesn't happen on your device. I will do some more searching.
